app/services/api/conversions/create_project_service.rb

```diff
@@ -1,26 +1,51 @@
 class Api::Conversions::CreateProjectService
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveRecord::AttributeAssignment
+
   class ProjectCreationError < StandardError; end
 
+  attribute :urn
+  attribute :incoming_trust_ukprn
+  attribute :advisory_board_date
+  attribute :advisory_board_conditions
+  attribute :provisional_conversion_date
+  attribute :directive_academy_order
+  attribute :created_by_email
+  attribute :created_by_first_name
+  attribute :created_by_last_name
+
+  validates :urn, presence: true, urn: true
+  validate :establishment_exists
+  validates :incoming_trust_ukprn, presence: true, ukprn: true
+  validate :trust_exists
+
   def initialize(project_params)
     @project_params = project_params
```

Comment:
I dont' think we need this? calling super assigns all the attributes and then we can just get them via their getters later?